lib/pundit.rb

```diff
@@ -56,11 +56,19 @@ def policy!(user, record)
     end
   end
 
+  module Helper
+    def policy_scope(scope)
+      policy_scopes[scope] ||= Pundit.policy_scope!(pundit_user, scope)
+    end
+  end
+
   included do
+    helper Helper if respond_to?(:helper)
     if respond_to?(:helper_method)
-      helper_method :policy_scope
       helper_method :policy
       helper_method :pundit_user
+      helper_method :policies
```

Comment:
I only did that because it seemed inconsistent that `policy_scopes` would be a helper method and `policies` would not.